docs/products/cassandra/howto/manage-cross-cluster-replication.md

```diff
@@ -1,33 +1,21 @@
 ---
 title: Manage cross-cluster replication in Aiven for Apache Cassandra®
+pro: true
```

Comment:
Already raised it on slack, but just for being extra sure! Pro Platform is now Aiven Entreprise